minard/views.py

```diff
@@ -1296,6 +1318,33 @@ def pca_run_detail(run_number):
                            run_number=run_number,
                            run=run)
 
+@app.route('/burst_run_detail/<run_number><subrun><sub>')
+def burst_run_detail(run_number, subrun, sub):
+    server = couchdb.Server("https://snoplus:"+app.config["COUCHDB_PASSWORD"]+"@"+app.config["COUCHDB_HOSTNAME"])
+    db = server["burst"]
+
+    for row in db.view('_design/burst/_view/burst_by_run'):
+        if (int(row.key[0]) == int(run_number)):
+            if (int(row.key[1]) == int(subrun)):
+                if (int(row.key[2]) == int(sub)):
+                    run_id = row.id
+                    try:
+                        result = dict(db.get(run_id).items())
+                    except KeyError:
+                        app.logger.warning("Code returned KeyError searching for burst_details information in the couchDB. Run Number: %d" % run_number)
+                    subrun = str(result['subrun'])
+                    sub = str(result['burst_n'])
+                    files = str(run_number+"_"+subrun+"_"+sub)
```

Comment:
Could this be written as:

```python
files = "%i_%i_%i" % (run,subrun,sub)
```

and then you can delete the two conversions above?